beacon_node/http_api/src/block_id.rs

```diff
@@ -250,6 +251,37 @@ impl BlockId {
             }
         }
     }
+
+    /// Return the `BlobSidecarList` identified by `self`.
+    pub async fn blob_sidecar_list<T: BeaconChainTypes>(
+        &self,
+        chain: &BeaconChain<T>,
+    ) -> Result<BlobSidecarList<T::EthSpec>, warp::Rejection> {
+        let root = self.root(chain)?.0;
+        chain
+            .get_blobs(&root)
+            .map_err(warp_utils::reject::beacon_chain_error)
+    }
+
+    pub async fn blob_sidecar_list_filtered<T: BeaconChainTypes>(
```

Comment:
Nor does this one.

Reply:
removed in same commit as above: 5f958781a0b922739e18eefab52eee4b4ad8d13b